src/renderer/components/App/AppIntro/AppIntroScreen.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="flex flex-row w-full h-full">
+    <div class="AppIntroScreen__container__left flex-1 animated fadeIn rounded-lg">
+      <div
+        v-if="showLogo"
+        class="AppIntroScreen__container__left__logo"
+      >
+        <img
+          :src="assets_loadImage('ark-logo.png')"
+          class="h-full"
+        >
+      </div>
+      <div
+        :class="showLogo ? 'h-3/5 w-3/5 flex ' : 'h-full w-full'"
+        class="h-full w-full flex items-center justify-center"
+      >
+        <img
+          :class="showLogo ? 'h-3/5 w-3/5' : 'h-full w-full'"
+          :src="assets_loadImage(image)"
+        >
+      </div>
+    </div>
+
+    <div class="AppIntroScreen__container__right flex-1 flex-col align-center justify-center h-full w-full ml-4 bg-theme-feature rounded-lg">
+      <div
+        class="flex flex-col items-center h-full w-full animated fadeIn font-medium"
+        :class="showFooter ? 'justify-between' : 'justify-center'"
+      >
+        <slot name="content" />
+
+        <slot
+          v-if="showFooter"
+          name="buttons"
+        >
+          <div class="flex flex-row w-full justify-between px-16 pb-16 sm:px-10 sm:pb-10">
```

Comment:
The initial style should be switched, and replicated to the parent elements.

```suggestion
          <div class="flex flex-row w-full justify-between px-10 pb-10 md:px-16 md:pb-16">
```

Reply:
I've removed this; since now the left part (the image) is removed, this trick isn't necessary